src/api/spec/models/obs_factory/distribution_strategy_factory_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ObsFactory::DistributionStrategyFactory do
+  let(:project) { create(:project) }
+
+  describe '::new' do
+    context 'called with no parameters' do
+      subject { ObsFactory::DistributionStrategyFactory.new }
+
+      it { is_expected.to be_kind_of(ObsFactory::DistributionStrategyFactory) }
```

Comment:
What are you testing with this? `Class.new`?

Reply:
Just have a look some lines [above](https://github.com/openSUSE/open-build-service/pull/5191/files#diff-ad0180b8e7006a09dab526848e938a24R6) :wink: